jina/orchestrate/deployments/__init__.py

```diff
@@ -318,19 +323,6 @@ def update_pod_args(self):
         else:
             self.pod_args = self._parse_args(self.args)
 
-        if self.external:
```

Comment:
we decided to move it here, so that all setting logic is in one place
<img width="524" alt="image" src="https://user-images.githubusercontent.com/15269265/207570433-f117ca6e-c3ec-4c00-97f7-6f4bae0ce842.png">

Reply:
okey perfect